src/components/DashboardHeader/DashboardHeader.types.ts

```diff
@@ -2,4 +2,5 @@ export interface DashboardHeaderProps {
   userAvatar?: string;
   userName?: string;
   logo: string;
+  userRole?: string;
```

Comment:
For now this can stay but once we implement login mechanism we will fetch all these information from custom hook